setup.py

```diff
@@ -11,7 +11,7 @@
 setup(
     name='pypodcastparser-ihr',
 
-    version='1.3.0',
+    version='1.4.3-dev',
```

Comment:
Change this before me merge.

Reply:
👍

Reply:
upgraded to 1.5.0. will need to in ep ingestion too